src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import core.basesyntax.reader.CsvFileReader;
+import core.basesyntax.report.CsvFileWriter;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        CsvFileReader csvFileReader = new CsvFileReader("testData1.csv");
```

Comment:
Avoid hardcoding file paths in your service class constructors. It's better to pass the file path as a parameter to the method that needs it.